.github/workflows/test-harness-acapy-anoncreds.yml

```diff
@@ -1,21 +1,4 @@
-name: test-harness-acapy-acapy
-# RUNSET_NAME: "ACA-PY to ACA-Py"
```

Comment:
I believe the comments you removed are being pulled in by a script the generates the gh-pages published by this repo.  Suggesting putting them back and updating them to be accurate vs. deleting them.

Reply:
Returned.